src/components/v1/scope/controllers/index.js

```diff
@@ -16,7 +16,16 @@ const getScopeById = async (req, res) => {
   });
 };
 
+const getScopes = async (req, res) => {
+  res.status(200).json({
+    status: 200,
+    message: 'Success',
+    data: await scopeServices.getScopes(req.query),
+  });
```

Comment:
@k4dm3l  Where do you return the count property?

Reply:
the service return an object with 2 properties, one of that is called **records**

![image](https://user-images.githubusercontent.com/46683077/132409007-f17229fe-ae65-4096-a086-0e47daf666ec.png)